sonic_platform_base/sonic_xcvr/api/public/cmisCDB.py

```diff
@@ -415,6 +415,7 @@ def validate_fw_image(self):
         cmd = bytearray(b'\x01\x07\x00\x00\x00\x00\x00\x00')
         cmd[133-INIT_OFFSET] = self.cdb_chkcode(cmd)
         self.write_cdb(cmd)
+        time.sleep(2)
```

Comment:
@stephenxs instead of fixing just the firmware download is it better to fix the cdb1_chkstatus()? I see that `cdb1_chkstatus()` already has 60 seconds of wait time while checking CDB status. if there are i2c read errors then that will be considered as CDB is busy? So Why we need this fix here?

Reply:
@prgeor It retries in CDBCheckstatus after getting CDB status, like busy. But the I2C error can occur while reading CDB status.
We wait for an extra 2 second delay, just to make sure the cable is able to provide the status.
Make sense?